### ISSUE TITLE
2019-12-06 M1SS - remove on parameter at a time from PARAMETERLIST to identify sensitive parameters for which proper dependency management is not occurring leading to crashes

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#88]
Parameters such as CLAST, TLAST, AUCINFO are the most frequently identified base parameters. In some cases kel_v is not managed with appropriate dependency management.

```
** from 2019-12-06_M1SS_1TAUTOLD.json
mct$PARAMETERLIST <- paste(c("AUCALL","AUCALLDN","AUCDN","AUCINFO","AUCINFOC","AUCINFODN","AUCINFOi","AUCINFP","AUCINFPC","AUCINFPDN","AUCINFPi","AUCLAST","AUCLASTi","AUCLASTC","AUCLASTCi","AUCLASTDN","AUCLASTDNi","AUCT","AUCT1_T2","AUCTAUDNi","AUCTAU","AUCTAUi","AUCTDN","AUCXPCTO","AUCXPCTOi","AUCXPCTP","AUCXPCTPi","AUMCINFO","AUMCINFOi","AUMCINFP","AUMCINFPi","AUMCLAST","AUMCLASTi","AUMCTAUi","AUMCXPTO","AUMCXPTOi","AUMCXPTP","AUMCXPTPi","CAVi","CEST","CLAST","CLASTi","CLFO","CLFOW","CLFP","CLFPW","CLFTAUi","CLFTAUWi","CMAX","CMAXi","CMAXDN","CMAXDNi","CMIN","CMINi","CMINDN","CMINDNi","CTROUGHi","CTROUGHENDi","DIi","DOSE","DOSEi","DOSEC","DOSECi","F","FREL","FRELLAST","FRELLASTi","FTAUi","KEL","KELC0","KELNOPT","KELR","KELRSQ","KELRSQA","KELTMHI","KELTMLO","LASTTIME","LASTTIMEi","MRTEVIFO","MRTEVIFOi","MRTEVIFP","MRTEVIFPi","MRTLAST","MRTLASTi","PTF","PTFi","PTR","PTRi","PTROUGHRi","PTROUGHRENDi","RACi","RACCMAXi","RACCMINi","RSSi","TAU","TAUi","THALF","THALFF","TLAG","TLAST","TLASTi","TMAX","TMAXi","TMIN","TMINi","TOLD","TOLDi","VZFO","VZFOW","VZFP","VZFPW","VZFTAUi","VZFTAUWi","FLGACCEPTPREDOSE","FLGACCEPTTMAX","FLGACCEPTKEL","FLGACCEPTTAU"), collapse=";")

*** i:  AUCTAUi  removed
[1] "AUCTDN;AUCXPCTO;AUCXPCTOi;AUCXPCTP;AUCXPCTPi;AUMCINFO;AUMCINFOi;AUMCINFP;AUMCINFPi;AUMCLAST;AUMCLASTi;AUMCTAUi;AUMCXPTO;AUMCXPTOi;AUMCXPTP;AUMCXPTPi;CAVi;CEST;CLAST;CLASTi;CLFO;CLFOW;CLFP;CLFPW;CLFTAUi;CLFTAUWi;CMAX;CMAXi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;F;FREL;FRELLAST;FRELLASTi;FTAUi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;RSSi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFO;VZFOW;VZFP;VZFPW;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU"
unit_conversion : Unit Class 1 (Time) time_col:  TMAX TMAX1 TMAX2 TMIN TMIN1 TMIN2 TLAST TLAST1 TLAST2 TLAG KELTMLO KELTMHI THALF LASTTIME LASTTIME1 LASTTIME2 MRTLAST MRTLAST1 MRTLAST2 MRTEVIFO1 MRTEVIFO2 MRTEVIFP1 MRTEVIFP2 TAU1 TAU2 TOLD1 TOLD2  parameters are scaled from  HR  to  HR  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE1  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 3 (Dose) dose_col:  DOSE2  parameters are scaled from  MG  to  MG  via scaling factor:  1 
unit_conversion : Unit Class 4 (Volume) volume_col:  VZFTAU1 VZFTAU2  parameters are scaled from  ML  to  L  via scaling factor:  0.001 
unit_conversion : Unit Class 5 (Amount/Volume) conc_col:  CMAX CMAX1 CMAX2 CMIN CMIN1 CMIN2 CLAST CLAST1 CLAST2 CTROUGH1 CTROUGH2 CTROUGHEND1 CTROUGHEND2 KELC0 CEST CAV1 CAV2  parameters are scaled from  UG/ML  to  NG/ML  via scaling factor:  1000 
unit_conversion : Unit Class 6 (1/Time) kel_col:  KEL  parameters are scaled from  1/HR  to  1/HR  via scaling factor:  1 
unit_conversion : Unit Class 7 (Volume/Time) cl_col:  CLFTAU1 CLFTAU2  parameters are scaled from  ML/HR  to  L/HR  via scaling factor:  0.001 
Error in unit_conversion(data = data_data, map = map_data, result = computation_df,  : 
  object 'auc_unit_tmp' not found
In addition: There were 50 or more warnings (use warnings() to see the first 50)
> 

*** i:  CLAST  removed
[1] "CLASTi;CLFO;CLFOW;CLFP;CLFPW;CLFTAUi;CLFTAUWi;CMAX;CMAXi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;F;FREL;FRELLAST;FRELLASTi;FTAUi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;RSSi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFO;VZFOW;VZFP;VZFPW;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi"
Error in value[[3L]](cond) : 
  Error in auc_inf_o(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'c_last' not found
For SDEID 70745987
In addition: There were 50 or more warnings (use warnings() to see the first 50)


*** i:  CTROUGHi  removed
[1] "CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST"
Error in value[[3L]](cond) : 
  Error in ptroughr(cmax = c_maxi[[d]], ctrough = c_troughi[[d]]): object 'c_troughi' not found
For SDEID 70745987
In addition: There were 50 or more warnings (use warnings() to see the first 50)
> 

*** i:  CTROUGHENDi  removed
[1] "DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi"
Error in value[[3L]](cond) : 
  Error in ptroughrend(cmax = c_maxi[[d]], ctrough = c_troughendi[[d]]): object 'c_troughendi' not found
For SDEID 70745987
In addition: Warning messages:
1: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
2: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
3: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in ptroughrend(cmax = c_maxi[[d]], ctrough = c_troughendi[[d]]): object 'c_troughendi' not found
For SDEID 70745987 

*** i:  MRTEVIFOi  removed
[1] "MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi"
Error in value[[3L]](cond) : 
  Error in auc_XpctO(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'aucinf_o' not found
For SDEID 70745987
In addition: There were 50 or more warnings (use warnings() to see the first 50)

*** i:  MRTLASTi  removed - why is AUCINFO computer here?
[1] "PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi"
Error in value[[3L]](cond) : 
  Error in auc_inf_o(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'auclast' not found
For SDEID 70745987
In addition: There were 44 warnings (use warnings() to see them)

*** i:  TLAST  removed - why is AUCLAST neing computed here
[1] "TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi"
Error in value[[3L]](cond) : 
  Error in auc_last(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 't_last' not found
For SDEID 70745987
In addition: There were 50 or more warnings (use warnings() to see the first 50)

*** i:  VZFTAUi  removed
[1] "VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi;TLAST"
Error in value[[3L]](cond) : 
  Error in vzftau(kel = kel_v[["KEL"]], auctau = auctau[[d]], dose = dose_ci[[d]]): object 'dose_ci' not found
For SDEID 70745987
In addition: There were 50 or more warnings (use warnings() to see the first 50)

*** i:  FLGACCEPTTMAX  removed
[1] "FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi;TLAST;VZFTAUi"
Error in value[[3L]](cond) : 
  Error in aumc_lin_log(conc = conc, time = time, exflag = exflag, t_max = t_max): object 't_maxi' not found
For SDEID 70745987
In addition: There were 33 warnings (use warnings() to see them)

*** i:  CLAST  removed
[1] "CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi;TLAST;VZFTAUi;FLGACCEPTTMAX"
Error in value[[3L]](cond) : 
  Error in auc_inf_o(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'c_last' not found
For SDEID 70745987
In addition: There were 45 warnings (use warnings() to see them)

*** i:  MRTLASTi  removed
[1] "TLAST;VZFTAUi;FLGACCEPTTMAX;CLAST"
Error in value[[3L]](cond) : 
  Error in auc_inf_p(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 'auclast' not found
For SDEID 70745987
In addition: There were 45 warnings (use warnings() to see them)

*** i:  TLAST  removed
[1] "VZFTAUi;FLGACCEPTTMAX;CLAST;MRTLASTi"
Error in value[[3L]](cond) : 
  Error in auc_last(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 't_last' not found
For SDEID 70745987
In addition: Warning messages:
1: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'KELRSQ'
2: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'KELNOPT'
3: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
4: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
5: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTPREDOSECRIT' cannot be computed if 'CMAXi' is not part of the calculated parameters
6: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in auc_last(conc = tmp_df[, map_data$CONC], time = tmp_df[, map_data$TIME], : object 't_last' not found
For SDEID 70745987 

*** i:  VZFTAUi  removed
[1] "FLGACCEPTTMAX;CLAST;MRTLASTi;TLAST"
Error in value[[3L]](cond) : 
  Error in grep(x, parameter_list, ignore.case = TRUE, perl = TRUE): object 'kel_v' not found
For SDEID 70745987
In addition: Warning messages:
1: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'KELRSQ'
2: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'KELNOPT'
3: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTKELCRIT' values provided via 'map' does not have a valid parameter name 'AUCXPCTP'
4: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGEMESIS' is not present in the dataset
5: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTPREDOSECRIT' cannot be computed if 'CMAXi' is not part of the calculated parameters
6: In run_M1_SS_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : 
  Error in grep(x, parameter_list, ignore.case = TRUE, perl = TRUE): object 'kel_v' not found
For SDEID 70745987 
```

**Comment by jhhughes256** [comment]
Update
Testing results for above examples as of commit fce7a3c. All testing was done with tc071 M1SS. Tick represents success (i.e. error can no longer be replicated).

- [ ] AUCTAUi removed "AUCTDN;AUCXPCTO;AUCXPCTOi;AUCXPCTP;AUCXPCTPi;AUMCINFO;AUMCINFOi;AUMCINFP;AUMCINFPi;AUMCLAST;AUMCLASTi;AUMCTAUi;AUMCXPTO;AUMCXPTOi;AUMCXPTP;AUMCXPTPi;CAVi;CEST;CLAST;CLASTi;CLFO;CLFOW;CLFP;CLFPW;CLFTAUi;CLFTAUWi;CMAX;CMAXi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;F;FREL;FRELLAST;FRELLASTi;FTAUi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;RSSi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFO;VZFOW;VZFP;VZFPW;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU"
- [ ] CLAST removed "CLASTi;CLFO;CLFOW;CLFP;CLFPW;CLFTAUi;CLFTAUWi;CMAX;CMAXi;CMAXDN;CMAXDNi;CMIN;CMINi;CMINDN;CMINDNi;CTROUGHi;CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;F;FREL;FRELLAST;FRELLASTi;FTAUi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;RSSi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFO;VZFOW;VZFP;VZFPW;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi"
- [ ] CTROUGHi removed "CTROUGHENDi;DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST"
- [ ] CTROUGHENDi removed "DIi;DOSE;DOSEi;DOSEC;DOSECi;KEL;KELC0;KELNOPT;KELR;KELRSQ;KELRSQA;KELTMHI;KELTMLO;LASTTIME;LASTTIMEi;MRTEVIFO;MRTEVIFOi;MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi"
- [ ] MRTEVIFOi removed "MRTEVIFP;MRTEVIFPi;MRTLAST;MRTLASTi;PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi"
- [ ] MRTLASTi  removed - why is AUCINFO computer here? "PTF;PTFi;PTR;PTRi;PTROUGHRi;PTROUGHRENDi;RACi;RACCMAXi;RACCMINi;TAU;TAUi;THALF;THALFF;TLAG;TLAST;TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi"
- [ ] TLAST  removed - why is AUCLAST neing computed here "TLASTi;TMAX;TMAXi;TMIN;TMINi;TOLD;TOLDi;VZFTAUi;VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi"
- [ ] VZFTAUi removed
"VZFTAUWi;FLGACCEPTPREDOSE;FLGACCEPTTMAX;FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi;TLAST"
- [ ] FLGACCEPTTMAX  removed "FLGACCEPTKEL;FLGACCEPTTAU;AUCTAUi;CLAST;CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi;TLAST;VZFTAUi"
- [ ] CLAST  removed "CTROUGHi;CTROUGHENDi;MRTEVIFOi;MRTLASTi;TLAST;VZFTAUi;FLGACCEPTTMAX"
- [ ] MRTLASTi  removed "TLAST;VZFTAUi;FLGACCEPTTMAX;CLAST"
- [ ] TLAST  removed "VZFTAUi;FLGACCEPTTMAX;CLAST;MRTLASTi"
- [ ] VZFTAUi  removed "FLGACCEPTTMAX;CLAST;MRTLASTi;TLAST"

**Comment by rudraya** [comment]
This issue was addressed in a377a56, please let me know if it still occurs. I tested again using the parameters list provided in the above comment for when:
- AUCTAUi removed
- CTROUGHi removed
- CTROUGHENDi removed
- VZFTAUi removed

**Comment by jhhughes256** [comment]
### Update

As of a377a56 all test cases listed here run without computation error.
